WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/PrivateCoinJoinProfileViewModel.cs

```diff
@@ -43,7 +43,7 @@ public override bool Equals(object? obj)
 			return false;
 		}
 
-		return profile.AnonScoreTarget < MaxAnonScore && profile.AnonScoreTarget > MinAnonScore && profile.FeeRateMedianTimeFrameHours == FeeRateMedianTimeFrameHours && profile.RedCoinIsolation == RedCoinIsolation;
+		return profile.AnonScoreTarget < MaxAnonScore && profile.AnonScoreTarget >= MinAnonScore && profile.FeeRateMedianTimeFrameHours == FeeRateMedianTimeFrameHours && profile.RedCoinIsolation == RedCoinIsolation;
```

Comment:
```suggestion
		return profile.AnonScoreTarget <= MaxAnonScore && profile.AnonScoreTarget >= MinAnonScore && profile.FeeRateMedianTimeFrameHours == FeeRateMedianTimeFrameHours && profile.RedCoinIsolation == RedCoinIsolation;
```

Reply:
shouldn't this also be for MaxAnonScore?

Reply:
MaxAnonScore is `101`, so no, I don't think so.